### ISSUE TITLE
[Bug]: umweltverbaende_at stopped working/new website

### ISSUE THREAD

**Issue by thargor** [mampfes/hacs_waste_collection_schedule#3090]
### I Have A Problem With:

A specific source

### What's Your Problem

umweltverbaende_at https://xxxx.umweltverbaende.at got a new wordpress based website and it stopped working.

A new scraper needs to be implemented.

### Source (if relevant)

umweltverbaende_at

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [ ] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by thargor** [comment]
I tried to implement a new source but it seems not all subpages of umweltverbaende use the new software version. from the test cases only https://hollabrunn.umweltverbaende.at fails

**Comment by 5ila5** [comment]
I added a new parser for the new website, but it should properly fall back to the old approach if it detects, that it's an old page. Some of the old ones work right, while some seem to no longer have a collection calendar. But there is nothing to do about that.

Added in #3334


This will be part of the next release or is available now as master (git) version 

install master/git: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA


<details><summary>Older versions of HACS:</summary>
<p>
install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
</p>
</details>
